Feature/movies work

rhiannonlynne opened the pull request:
Mostly changes to make the movies work again. But also a collection of other updates and fixes that went into v4 and now being cherry-picked back to master.

yoachim commented:
The first example call:
`python example_movie.py opsimblitz2_1060_sqlite.db --ips 2`
dies because of some filename mis-matching (the fileroot isn't picking up the extra stuff that gets added from the sql constraint). I tried swaping in glob, but that broke other things downstream.

yoachim commented:
zomg this is painfully slow. Did I never write a way to convert accumulate metrics to movies? `mkDualMovies.sh` is doing bad loops, hit me with a jira ticket and I'll cook up an example that should be much faster.

yoachim commented:
Using the `mkDualMovies.sh`, looks like more string/path issues:
`Could find no file with path 'minion_1016_cumulative/minion_1016_Coaddm5Metric_HEAL_%024d_SkyMap.png' and index in the range 0-4
minion_1016_cumulative/minion_1016_Coaddm5Metric_HEAL_%024d_SkyMap.png: No such file or directory`

rhiannonlynne commented:
Uhh. I have no idea what happened to the history here. 
I fixed the problems here, but had a conflict with the plot Handler. To fix the conflict, I rebased on master. Looks like it was okay, but now the history of changes here is a mess. (?)

No conflicts, all works.
